### PR TITLE
Change the default file in the FileViewer if webextension (issue #1086)

### LINKED ISSUE #1086: Clicking on Contents for a webextension loads a 404
## steps to reproduce
1. Submit a web extension to AMO
2. Go to Editor tools and navigate to the reviewer details page for that add-on
3. Click on the 'Contents' link for that add-on

## observed behavior
https://addons-dev.allizom.org/en-US/firefox/files/browse/247966/ is a 404
https://addons-dev.allizom.org/en-US/editors/review/sort-tabs-by-url11 is a 404

## Notes:
Link for older extensions work fine.

### PR DESCRIPTION
**Pull request by magopian**
Fixes #1086

### CODE REVIEW

File: apps/files/helpers.py [hunk 1]
```diff
@@ -197,14 +197,14 @@ def is_directory(self):
...
-        if self.is_search_engine() and not key:
-            files = self.get_files()
-            return files.keys()[0] if files else None
-
         if key:
             return key
 
-        return 'install.rdf'
+        files = self.get_files()
+        for manifest in ['install.rdf', 'manifest.json', 'package.json']:
```
Comment (EnTeQuAk) [inline apps/files/helpers.py]: Nitpick: this could actually be a tuple.

File: apps/files/tests/test_helpers.py [hunk 2]
```diff
@@ -201,7 +201,18 @@ def test_contents_size(self):
...
+
+    def test_default_webextension(self):
+        viewer = FileViewer(make_file(2, get_file('webextension.xpi')))
+        viewer.extract()
+        assert viewer.get_default(None) == 'manifest.json'
+
+    def test_default_package_json(self):
+        viewer = FileViewer(make_file(3, get_file('new-format-0.0.1.xpi')))
+        viewer.extract()
+        assert viewer.get_default(None) == 'package.json'
```
Comment (EnTeQuAk) [inline apps/files/tests/test_helpers.py]: There is no test for the `else None` part of the return statement in case none of the files were found.

**Comment by EnTeQuAk** [comment]
r+wc

File: apps/files/helpers.py (hunk 1 above)
Comment (magopian) [inline apps/files/helpers.py]: Indeed, thanks ;)

File: apps/files/tests/test_helpers.py (hunk 2 above)
Comment (magopian) [inline apps/files/tests/test_helpers.py]: There's already one in another TestCase: https://github.com/magopian/olympia/blob/1086-change-default-file-in-viewer/apps/files/tests/test_helpers.py#L260, what do you think?

Comment (EnTeQuAk) [inline apps/files/tests/test_helpers.py]: Ah, apologies, I'm blind.

Comment (magopian) [inline apps/files/tests/test_helpers.py]: I wasn't part of the diff, so easily missed :)

**Pull request merged**